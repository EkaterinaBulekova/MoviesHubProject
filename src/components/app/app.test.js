import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import App from './app';

jest.mock('../../utils/data-provider/data-provider.js');
describe('<App />', () => {
  it('should render as expect', () => {
    const component = shallow(<App/>);
    expect(toJson(component)).toMatchSnapshot();
  });
});;
