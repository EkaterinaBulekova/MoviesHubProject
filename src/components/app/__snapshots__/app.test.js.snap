// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> should render as expect 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;
